### PR TITLE
Fix commonServerPython date_to_timestamp

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14442

## Description
Fixes an issue with date_to_timestamp function commonServerPython, and added test to that function.

## Must have
- [X] Tests
- [ ] Code Review

### CODE REVIEW

File: TestPlaybooks/script-TestPyCommonServer.yml [hunk 1]
```diff
@@ -337,6 +337,10 @@ script: |-
       assert camelize([{'chookity_bop': 'asdasd'}, {'ab_c': 'd e', 'fgh_ijk': 'lm', 'nop': 'qr_st'}], '_').__str__ == "[{u'ChookityBop': 'asdasd'}, {u'AbC': 'd e', u'Nop': 'qr_st', u'FghIjk': 'lm'}]"
       assert camelize({'ab_c': 'd e', 'fgh_ijk': 'lm', 'nop': 'qr_st'}, '_').__str__ == "{u'AbC': 'd e', u'Nop': 'qr_st', u'FghIjk': 'lm'}"
 
+  def test_date_to_timestamp():
```
Comment (ronykoz) [inline TestPlaybooks/script-TestPyCommonServer.yml]: I think that it should be mentioned in the TESTS dict in order to be tested, Also saw that CAMELIZE is not there so please add both of them

File: Tests/conf.json [hunk 2]
```diff
@@ -2,6 +2,10 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": [],
```
Comment (ronykoz) [inline Tests/conf.json]: Could you remove this line, not needed

File: TestPlaybooks/script-TestPyCommonServer.yml (hunk 1 above)
Comment (DeanArbel) [reply TestPlaybooks/script-TestPyCommonServer.yml]: Done

File: Tests/conf.json (hunk 2 above)
Comment (DeanArbel) [reply Tests/conf.json]: Done

**Pull request merged**